### ISSUE TITLE
Albert's Task 8/21/2023 - 8/25/2023

### ISSUE THREAD

**Issue by natt96z** [natt96z/cybersac#96]
- Make mock lesson videos with OBS/Video editing (Work in Progress/Linux)
- Finish CyberForensics VM with things we have created (w/Albert)
- Any Questions? Create quiz questions for students 
- Zoom Meeting With Group (Link provided on Fluffy Messaging)

-------------------------------------------------------------------------------------------------------------------------
Use OBS Screen Recorder or Screen-O-Matic for video recording opening up a Windows 10 Virtual Machine on provided laptops. 
Questions for students can be based on the site questions created, (But should be based on Windows)

**Comment by natt96z** [comment]
![image](https://github.com/natt96z/cybersac/assets/112590005/72695f0f-92c9-465a-9f2a-e584b022b9c2)
![vulnhub-walkthrough-18](https://github.com/natt96z/cybersac/assets/112590005/352f24d2-5882-4cac-883f-f52f4518cdf4)

![LINUXubunu](https://github.com/natt96z/cybersac/assets/112590005/688d3d37-beb0-451f-9cc2-0dc692347eeb)
![image](https://github.com/natt96z/cybersac/assets/112590005/b11f0905-934e-486f-836a-f7cad59aa879)


**_CyberForensics VM 8/21/2023 - 8/22/2023:_** We tested out the setup process of one of the VulnHub Virtual Machines, this one was called deception. Apparently it works as more of a network challenge. Only issue is the password locking on the virtual machine, me and Nate were unable to fix this issue but it will be fixed soon. netdiscovery and nmap is used in this virtual machines exercise.

**Comment by natt96z** [comment]
![261669012-dee994a5-c61f-4836-bb30-3c8a64658527](https://github.com/natt96z/cybersac/assets/112590005/044f947a-321f-4018-a52f-08e223410948)

Thursday Zoom Meeting With Group On Zoom (8/24/2023)(Group): During our group meeting, we delved into our strategy for conducting joint school visits. We strategized scripting our interactions, honing our communication skills, and reflecting on our previous observations from schools we had contacted about a month ago. This meeting proved pivotal in identifying additional tasks on our weekly agenda, such as VM work and collaborating on designing flyers. Patrice diligently documented the meeting minutes, capturing further details. Subsequently, there was a discussion with Nate regarding code cleanup later in the day.
